analytics/terraform/datahub-on-eks/datahub.tf

```diff
@@ -0,0 +1,10 @@
+module "datahub" {
+  depends_on            = [module.eks, module.eks_blueprints_kubernetes_addons]
+  source                = "./datahub-eks"
```

Comment:
rename this folder from `datahub-eks` to `datahub`. We already have top level folder saying datahub-on-eks

Reply:
still getting error when the folder/module name is the same as the Chart name.  Changed the folder name to datahub-addon.